docs/guides/deployment.md

```diff
@@ -1,2 +1,158 @@
+### Manifest
+**Prerequisite**
+- Kubernetes cluster setup
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) installed
+
+**Creating Kubernetes Resources**
+You need at least 2 mainfest for Raccoon. For [deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment) and for [configmap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/). Prepare both manifest as yaml file. You can fill the configuration as needed.
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: raccoon-config
+  namespace: default
+  labels:
+    application: raccoon
+data:
+  METRIC_STATSD_ADDRESS: "host.docker.internal:8125"
+  PUBLISHER_KAFKA_CLIENT_BOOTSTRAP_SERVERS: "host.docker.internal:9093"
+  SERVER_WEBSOCKET_CONN_UNIQ_ID_HEADER: "x-user-id"
+  SERVER_WEBSOCKET_PORT: "8080"
+```
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: raccoon
+  labels:
+    application: raccoon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      application: raccoon
+  template:
+    metadata:
+      labels:
+        application: raccoon
+    spec:
+      containers:
+      - name: raccoon
+        image: "odpf/raccoon:latest"
+        imagePullPolicy: IfNotPresent
+        resources:
+            limits:
+              cpu: 200m
+              memory: 512Mi
+            requests:
+              cpu: 200m
+              memory: 512Mi
+        envFrom:
+          - configMapRef:
+              name: raccoon-config
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+      volumes:
+```
+
+Suppose you save them as `configmap.yaml` and `deployment.yaml`. The next step is to apply the manifests to the kube cluster using `kubectl` command.
+```sh
+$ kubectl apply -f configmap.yaml -f deployment.yaml
+```
+You'll find the resources are created. To see the status of the deployment you can run
+```sh
+# Check deployment status
+$ kubectl get deployment raccoon
+# Check configmap status
+$ kubectl get configmap raccoon-config
+```
+
+**Configuration**
+You can add or modify the configurations inside `configmap.yaml` above. When you change the configmap, you also need to restart the deployment.
+
+### Helm
+**Prerequisite**
+- Kubernetes cluster setup
+- Helm installed
+
+Raccoon has helm chart maintained on [different repository](https://github.com/odpf/charts/tree/main/stable/raccoon). Refer to the repository for complete deployment guide.
+
+## Production Checklist
```

Comment:
Also include main config parameters,
KAFKA brokers address, topic pattern, server listening port, statsd host/port  etc. 
basically key config that will be needed in a production deployment.